src/magic.h

```diff
@@ -0,0 +1,290 @@
+#pragma once
+#ifndef MAGIC_H
+#define MAGIC_H
+
+#include "damage.h"
```

Comment:
To fix the `spellbook`.

```suggestion
#include <map>

#include "damage.h"
```